### PR TITLE
16.04

### PR DESCRIPTION
**Pull request by Mnakagawa**

### CODE REVIEW

**Comment by Mnakagawa** [comment]
@cosmo0920 @kz0817 @tksmatsunaga  
Please review!

File: contents/docs/install/16.04/en/index.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+For example, if the Hatohol Web Frontend runs on computer: 192.168.1.1,
+open the following URL from your Browser.
+
+- http://192.168.1.1/hatohol
+
+> ** Note **
+> Currently the above pages have been checked with Google Chrome and Firefox.
+> When using Internet Explorer, display layouts may collapse depending on the version of use.
+
+### Set up Hatohol Arm Plugin Interface 2 setting
```
Comment (cosmo0920) [inline contents/docs/install/16.04/en/index.md]: Hatohol 16.04から組み込みのZabbixやNagiosが消えているので If you want ... と言うよりもっと断定的に言っても良い気がします。

Comment (cosmo0920) [inline contents/docs/install/16.04/en/index.md]: HAP2プラグイン群を用いてZabbixやNagiosなどからデータを取得するには〜のドキュメントを参照して設定を行ってください。

みたいな説明文になりそうです。

Comment (cosmo0920) [inline contents/docs/install/16.04/en/index.md]: You should setup RabbitMQ and HAP2 to retrieve data from Zabbix, Nagios, and Ceilometer etc. via Hatohol Arm Plugin 2(HAP2).

みたいな感じですかね。

**Comment by cosmo0920** [comment]
Almost OK for me.
But if I were you, I'd write a note for ArmXXX which are built-in retrieving data mechanism had been removed.

**Comment by cosmo0920** [comment]
> But if I were you, I'd write a note for ArmXXX which are built-in retrieving data mechanism had been removed.

Only adding `This release contains breaking changes about several retrieving monitoring server mechanism. Please refer to the Hatohol 16.04 release note.`.

or, simply:

`Built-in Arm retrieving mechanism has been removed in this release`.

is better.

File: contents/docs/upgrade/16.04/en/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## ***16.04 doesn't support upgrade from previous version.***
+
+If you want to save data of previous version, you get DB backup by the following way, and import to DB in different name. Otherwise, don't import and keep it intact.
+
+## Removed ArmXXX
+
+Built-in Arm retrieving mechanism has been removed from hatohol server in this release.
+
+Please get use the monitoring data by Hatohol Arm Plugin2.
```
Comment (cosmo0920) [inline contents/docs/upgrade/16.04/en/index.md]: > Please get use the monitoring data by Hatohol Arm Plugin2. 

get useと動詞を続けて使うことはできないので、

From this release, please obtain the monitoring data with Hatohol Arm Plugin2.

**Comment by cosmo0920** [comment]
Any concerns?

If no concerns, :shipit:!

**Comment by Mnakagawa** [comment]
OK

**Pull request merged**